docs/rapha-rover/specification/ros-api.mdx

```diff
@@ -0,0 +1,578 @@
+
+  Reset motor faults and calculated consumed energy.
+
+- `controller/set_battery_modes` ([rapha_interfaces/srv/SetBatteryModes])
+
+  Set target battery modes.
+
+- `controller/set_steering_mode` ([rapha_interfaces/srv/SetSteeringMode])
+
+  Set the current steering mode.
+
+## Parameters
+
+### controller
+
+- `drivetrain.wheel_base` (type: `float`, default: `0.378`)
+
+  Distance between front and rear wheels. $[m]$
+
+- `drivetrain.track_width` (type: `float`, default: `0.384`)
+
+  Distance between left and right wheels. $[m]$
+
+- `drivetrain.wheel_radius` (type: `float`, default: `0.08`)
+
+  Radius of the wheel. $[m]$
+
+- `drivetrain.max_wheel_angular_velocity` (type: `float`, default: `19.0`)
+
+  Maximum allowable wheel angular velocity. $[\frac{rad}{s}]$
+
+- `drivetrain.change_mode_acceleration` (type: `float`, default: `2.0`)
+
+  Acceleration to use when changing steering mode.
+  $[\frac{m}{s^2}, \frac{rad}{s^2}]$
+
+- `drivetrain.change_mode_steering_angle_velocity` (type: `float`, default:
+  `4.0`)
+
+  Steering angle velocity to use when changing steering mode. $[\frac{rad}{s}]$
+
+- `drivetrain.max_steering_angle` (type: `float`, default: `1.08`)
+
+  Maximum steering angle used in ackermann mode. $[rad]$
+
+- `drivetrain.input_timeout` (type: `float`, default: `0.5`)
+
+  The duration from last received target, after which the controller will stop
+  the robot. Set to 0 to disable. $[s]$
+
+- `drivetrain.wheel_disable_timeout` (type: `float`, default: `10.0`)
+
+  The duration from last received target, after which the controller will
+  disable to motors. Set to 0 to disable. $[s]$
+
+- `drivetrain.servo_left_position_offset` (type: `float`, default: `0.0`)
+
+  Offset added to the left servo position. $[rad]$
+
+- `drivetrain.servo_right_position_offset` (type: `float`, default: `0.0`)
+
+  Offset added to the right servo position. $[rad]$
+
+- `drivetrain.servo_calibration_power_threshold` (type: `float`, default:
+  `20.0`)
+
+  The minimal motor power after which we assume the servo is pressing against
+  the bumper during calibration procedure. $[W]$
+
+- `drivetrain.servo.acceleration_divider` (type: `int`, default: `10`)
+
+  Raw acceleration divider value passed to servo motors.
+
+- `drivetrain.servo.position_loop_speed` (type: `float`, default: `5.0`)
+
+  Position loop speed value passed to servos motors. $[\frac{rad}{s}]$
+
+- `drivetrain.wheel.acceleration_divider` (type: `int`, default: `2`)
+
+  Raw acceleration divider value passed to wheel motors.
+
+{/* TODO: Other nodes parameters */}
+
+## Custom message types
+
+{/* TODO: Remove this section after rapha_interfaces message definitions start being hosted in docs.ros2.org */}
+
+- `rapha_interfaces/msg/SteeringMode`
+
+<a id="msg-steering-mode">
+
+```bash title="Represents the steering mode used by the drivetrain controller."
+uint8 ACKERMANN=0
+uint8 TURN_IN_PLACE=1
+
+uint8 data
+```
+
+</a>
```

Comment:
`<a>` html tags should be empty when specifying anchors. Current usage can lead to undesired results like wrong colour of the text.

Example fix:

```suggestion
<a id="msg-steering-mode"></a>

```bash title="Represents the steering mode used by the drivetrain controller."
uint8 ACKERMANN=0
uint8 TURN_IN_PLACE=1

uint8 data
```
```